molecule/provisioner/ansible/playbooks/docker/create.yml

```diff
@@ -22,11 +22,27 @@
         - item.registry.credentials is defined
         - item.registry.credentials.username is defined
 
+    - name: Check presence of custom Dockerfiles
+      stat:
+        path: "{{ molecule_scenario_directory + '/' + (item.dockerfile | default( 'Dockerfile.j2')) }}"
+      loop: "{{ molecule_yml.platforms }}"
+      register: dockerfile_stats
+
     - name: Create Dockerfiles from image names
+      vars:
+        # TODO(ssbarnea): expose module dir so it can also be used by plugins
```

Comment:
What about feeding it in from:

https://github.com/ansible/molecule/blob/97a7b60fc47b2b83f3f772aafbf044294be168a6/molecule/provisioner/ansible.py#L593-L601

Does it work?